toolkit/native-token-management/pallet/src/lib.rs

```diff
@@ -185,10 +225,19 @@ pub mod pallet {
 		}
 	}
 
+	#[pallet::hooks]
+	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
+		fn on_finalize(_block: BlockNumberFor<T>) {
+			TransferedThisBlock::<T>::set(None);
```

Comment:
```suggestion
			TransferedThisBlock::<T>::kill();
```
Would it work?

IDK why is it called `kill` instead of `empty` or `clear`.

Reply:
it does, ty